docs/plans/2026-06-02-report-viewer-ui-refresh-plan.md

```
@@ -0,0 +1,1324 @@
+Status: Historical
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)